runelite-client/src/test/java/net/runelite/client/plugins/chatnotifications/ChatNotificationsPluginTest.java

```diff
@@ -286,4 +289,48 @@ public void testHighlightOwnNameNbsp()
 		// set value uses our player name, which has nbsp replaced
 		verify(messageNode).setValue("<col=005f00><colHIGHLIGHT><u>Logic Knot</u><col=005f00> received a drop: Adamant longsword</col>");
 	}
-}
\ No newline at end of file
+
+	@Test
+	public void testLocalPlayerSelfMention()
+	{
+		final String localPlayerName = "Brooklyn";
+
+		MessageNode messageNode = mock(MessageNode.class);
+
+		Player localPlayer = mock(Player.class);
+		when(client.getLocalPlayer()).thenReturn(localPlayer);
+		when(localPlayer.getName()).thenReturn(localPlayerName);
+
+		lenient().when(config.highlightOwnName()).thenReturn(true);
+		lenient().when(config.notifyOnHighlight()).thenReturn(true);

```

Comment:
Neither test is specifically testing notifications; only the name/words highlight configs need to be mocked in the added tests.